src/gatk_gcnv_caller.sh

```
@@ -0,0 +1,655 @@
+    # prepare a batch_input string that has all sample_basecount.tsv file as an input
+    local batch_input
+    batch_input=$(find inputs/basecounts/ -type f -name '*_basecount.hdf5' -exec basename {} \; \
+        | sort | sed 's/^/--input \/data\/basecounts\//g')
+
+    SECONDS=0
+    docker run -v /home/dnanexus/inputs:/data \
+        "$GATK_image" gatk FilterIntervals \
+        -L /data/beds/preprocessed.interval_list \
+        -imr OVERLAPPING_ONLY \
+        --annotated-intervals /data/beds/annotated_intervals.tsv \
+        $batch_input $FilterIntervals_args \
```

Shellcheck

[warning] FilterIntervals_args is referenced but not assigned.

(SC2154)